### PR TITLE
[DOC] Revise step 2 input code example

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Corrects step input code example

### Issues Resolved
NA

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _ingest-pipelines/processors/ip2geo.md
Comment (heemin32) [inline _ingest-pipelines/processors/ip2geo.md:226]:
```suggestion
        "ip":"172.0.0.1",
```

**Comment by vagimeli** [comment]
@heemin32 Will you approve this PR? I can merge it once it's approved. Thank you :)

**Review by heemin32 — APPROVED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**
